docs/source/tutorial/starting.rst

```diff
@@ -56,6 +56,44 @@ install with the command ``make install``.  This will, by default, install into
 
     IDRIS2_PREFIX ?= /usr/local
 
+On Windows
+-------------------------
+
+On Windows systems you can build and use Idris 2 from sources using WSL (Windows Subsystem for Linux).
+
+Open Windows PowerShell as administrator and run:
+
+.. code-block:: PowerShell
+
+	wsl --instal # enable Window to run a linux system 
+	wsl --install -d Ubuntu # install a specific linux distro
+
+A restart may be required.
+
+Open PowerShell and run:
+
+.. code-block:: PowerShell
+
+	wsl -d Ubuntu # open a linux terminal
+	apt update # it is necessary to install dependencies
+	apt install make gcc chezscheme libgmp3-dev # install dependencies required for build from source
+	cd ~ # by default wsl will open in the current Windows directory, so we move to a directory inside the linux filesystem
+	git clone https://github.com/idris-lang/Idris2.git # download source code from github
+	cd Idris2 # move to the source code directory
+	make bootstrap SCHEME=chezscheme # builds the executable
+	make install # installs the executable
+	ln -s ~/.idris2/bin/idris2 /usr/local/bin/idris2 # add the binary to executable path
+	exit # exit the linux terminal 
+
+Open PowerShell in your source code directory and run:
+
+.. code-block:: PowerShell
+
+	wsl -d Ubuntu idris2 # to run idris2 command in the current Windows directory
+
+Or in alternative you can use vscode with wls and idris extensione combined.
```

Comment:
There are several extensions, which one do you reccomend? I think to link the "best" one as an example is a good idea

Reply:
Based off the number of downloads, this one? [https://marketplace.visualstudio.com/items?itemName=bamboo.idris2-lsp](https://marketplace.visualstudio.com/items?itemName=bamboo.idris2-lsp)

It also seems to be the recommended one from the other popular extension:

> For Idris 2, the language-server based extension provides additional features, such as semantic highlighting. If you are using the latest version of Idris 2, it may provide a better experience.

\- [maraymond2/idris-vscode:README](https://github.com/meraymond2/idris-vscode#which-extension-should-i-use)